src/dpos-user.ts

```diff
@@ -19,38 +20,13 @@ import {
 
 const log = debug('dpos-user')
 
-const coinMultiplier = new BN(10).pow(new BN(18))
-const V2_GATEWAYS = ['oracle-dev', 'asia1']
-
-const ERC20ABI = require('./mainnet-contracts/ERC20.json')
-const ValidatorManagerContractABI = require('./mainnet-contracts/ValidatorManagerContract.json')
-const ERC20GatewayABI = require('./mainnet-contracts/ERC20Gateway.json')
-const ERC20GatewayABI_v2 = require('./mainnet-contracts/ERC20Gateway_v2.json')
-
-import { ERC20 } from './mainnet-contracts/ERC20'
-import { ValidatorManagerContract } from './mainnet-contracts/ValidatorManagerContract'
-import { ERC20Gateway_v2 } from './mainnet-contracts/ERC20Gateway_v2'
 import { LocalAddress } from './address'
 import { getMetamaskSigner } from './solidity-helpers'
 
-enum GatewayVersion {
-  SINGLESIG = 1,
-  MULTISIG = 2
-}
+import { GatewayUser } from './gateway-user'
```

Comment:
These imports should all be up top, and there's already an import from `./gateway-user` up there